From d1d5dd354ad9118e4161ebf8009495627f9d0cde Mon Sep 17 00:00:00 2001
From: Han Gao <gaohan@iscas.ac.cn>
Date: Sun, 1 Sep 2024 02:05:22 +0800
Subject: [PATCH 294/400] dts: bt: support rtl8822cs

Configure the power ctrl pin for bt,and enable uart4

Signed-off-by: Han Gao <gaohan@iscas.ac.cn>
---
 arch/riscv/boot/dts/thead/th1520-rvbook.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/riscv/boot/dts/thead/th1520-rvbook.dtsi b/arch/riscv/boot/dts/thead/th1520-rvbook.dtsi
index 9904d67adfa6..98404c90181e 100644
--- a/arch/riscv/boot/dts/thead/th1520-rvbook.dtsi
+++ b/arch/riscv/boot/dts/thead/th1520-rvbook.dtsi
@@ -336,12 +336,11 @@ wcn_wifi: wireless-wlan {
  		status = "okay";
  	};
 
-	wcn_bt: wireless-bluetooth {
-		compatible = "bluetooth-platdata";
-		pinctrl-names = "default", "rts_gpio";
-		BT,power_gpio    = <&gpio0 22 GPIO_ACTIVE_HIGH>;
-		status = "okay";
-	};
+ 	wcn_bt: wireless-bluetooth {
+ 		compatible = "bluetooth-platdata";
+		BT,power-gpios = <&gpio0 22 GPIO_ACTIVE_HIGH>;
+ 		status = "okay";
+ 	};
 
 	gpio-keys {
 		compatible = "gpio-keys";
@@ -856,6 +855,7 @@ &uart3 {
 &uart4 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart4_pins>;
+	status = "okay";
 };
 
 &qspi1 {
-- 
2.43.0

